src/OrbitQt/orbitsamplingreport.cpp

```diff
@@ -93,7 +94,38 @@ void OrbitSamplingReport::Initialize(orbit_data_views::DataView* callstack_data_
     //  is no need for manual updates.
     orbit_data_views_.push_back(treeView);
 
-    QString thread_name = QString::fromStdString(report_data_view.GetName());
+    std::string thread_name_std = report_data_view.GetName();
+    QString thread_name = QString::fromStdString(thread_name_std);
```

Comment:
I would handle this a bit differently, especially since "thread name" is a bit confusing, this is the name of the tab and it contains the thread id, or "all threads".

I don't think it's a problem to call `report_data_view.GetName();` twice, so:
- Change `thread_name` to `tab_name` and declare it only before `addTab` with `QString::fromStdString(report_data_view.GetName());`
- Declare `thread_name_std` only if you need it before `notice_label`. Maybe call it differently, such as `tab_name_without_newline`. And, instead of using regexes, call `std::replace(<var_name>.begin(), <var_name>.end(), '\n', ' ');`. Then actually use it in for `notice_label`.

Let me know what you think.

Reply:
Sounds good. Done.